### ISSUE TITLE
Puppetmaster automaton is not really working

### ISSUE THREAD

**Issue by AuroraFlare** [DerpyProjectGroup/topaz#103]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/DerpyProjectGroup/topaz/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/DerpyProjectGroup/topaz/blob/info/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: FREEDOM

When using any maneuver for said attachment it just doesn't work or do anything.

Ranged Automaton does ranged damage for 0 damage. 

EDIT: Edited to clarify better of what's happening!

**Comment by claywar** [comment]
If the skills are misreported (assuming that its at 240), can you zone and confirm that its still displayed incorrectly?

**Comment by AuroraFlare** [comment]
I can say they are still showing incorrectly. 
![Sara_2021 05 11_194426](https://user-images.githubusercontent.com/11064151/117897882-b44ed780-b291-11eb-9150-08d7669dd187.png)

**Comment by claywar** [comment]
Assuming you have skilled Ranged for a SS body/head, then yes, that's absolutely an issue.  If there's another issue that you're experiencing, please clarify in the issue description.

**Comment by AuroraFlare** [comment]
I should also note the automaton will ranged attack for 0 points of damage on any mob and never skills up as it shows 0 for the skill. And well attachments are broken or maneuvers are.

**Comment by claywar** [comment]
Given the area that you're in with 0 ranged skill for the auto, that might not be unexpected (again, there still could be an issue, assuming the skill was leveled).  0-damage hits and misses are most likely not going to provide a skillup.

**Comment by AuroraFlare** [comment]
![Sara_2021 05 11_195719](https://user-images.githubusercontent.com/11064151/117898509-19ef9380-b293-11eb-8633-ab51ed0eb50f.png)
I also did a bumblebee and it still did 0 damage just to make sure.

**Comment by AuroraFlare** [comment]
I fixed the skill cap issue, I was looking into changing the skill levels, and that was 100% my bad. But I used a fresh install of the freedom branch and Ranged Attacks still do 0 damage and maneuvers do not work. My brother is a programmer so he helped me debug the ranged attacks and he said it shows up as a dagger skill (maybe this is correct) and the resists are all 100% for ranged attacks including the weapon skill. And maneuvers just do nothing. Hope this helps!
